.env.example

```diff
@@ -0,0 +1,3 @@
+DB_NAME=
+DB_USER=
+DB_PASSWORD=
```

Comment:
If we can set up different build environments (see comment on `config/settings.py`), let's add the `DATABASE_URL` variable here as well, please.